Assets/CritterController.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class CritterController : MonoBehaviour
+{
+    public float consumed, hydrated;
```

Comment:
`consumed` and `hydrated` both sound like names for booleans. I haven't looked at how they're used yet, but `consumption` and `hydration` might be better, esp. since `consumptionLostPerSecond` and `hydrationLostPerSecond` are things

I fully acknowledge that those variables are probably ones I chose in the processing version 😅